src/winter_onboarding_2021/fleet_management_service/views/layout.clj

```diff
@@ -8,6 +8,10 @@
    (include-css "/public/css/bootstrap.min.css")
    (include-js "/public/js/jquery-3.6.0.min.js")
    (include-js "/public/js/bootstrap.bundle.min.js")
+   [:script
+    {:defer "defer"
+     :async "async"
```

Comment:
Shall we consider not including the script in layout, since it will be loaded on every request whereas we only require in invites only.

Reply:
Please go through [this](https://developers.google.com/recaptcha/docs/display#:~:text=The%20script%20must%20be%20loaded%20using%20the%20HTTPS%20protocol%20and%20can%20be%20included%20from%20any%20point%20on%20the%20page%20without%20restriction). We can add the script in the signup form itself.